src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -1,13 +1,22 @@
-{% load i18n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
+{% load i18n l10n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
 
 <aside class="file" aria-label="{% trans "Bestand" %} {{ name }}">
     <div class="file__file">
```

Comment:
This needs a new class, so you can add the Red line on the left of this div, something like `{% if recently_added %}file__container--recent{% endif %}` and add a left-border as in the design.